### ISSUE TITLE
Video files not resizing properly on opera browser

### ISSUE THREAD

**Issue by aeonSolutions** [IQSS/dataverse.harvard.edu#283]
**What steps does it take to reproduce the issue?**
after uploading a video file to a dataset when opening the dataset page, the video file in the dataset 
on loading the webpage the video does not resize properly to match the iframe within the web page. As a consequence, it is impossible to view the video.,   Anyone can evaluate more on this at Harvard's Dataverse here:
https://doi.org/10.7910/DVN/0AXHEA

* When does this issue occur?
when I open a dataset with video files. 
when I want to view the video on the webpage itself. 

* Which page(s) does it occurs on?
in the dataset web page

* What happens?
The video file is not resized properly to match the iframe and webpage dimensions. 

* To whom does it occur (all users, curators, superusers)?
anyone with access. 

* What did you expect to happen?
I expect to view the video on the dataset web page without any effort. 

**Which version of Dataverse are you using?**
web version hosted at Harvard University. 

**Any related open or closed issues to this bug report?**

**Screenshots:**
![image](https://github.com/IQSS/dataverse/assets/7936768/ca56ae09-eb42-46a4-9022-0c32f086478c)

**Comment by cmbz** [comment]
2024/06/24
Transferred the issue, waiting on the next release to test and close.

**Comment by qqmyers** [comment]
[#61](https://github.com/gdcc/dataverse-previewers/pull/61) has been merged and that update is available in the betatest folder and the URL https://gdcc.github.io/dataverse-previewers/previewers/betatest/VideoPreview.html now. (The updated previewer can also be installed locally or one could add the 1 line in the css file from #61 into an existing local install.) So a previewer release is not a technical blocker.

W.r.t. policy/process - there's nothing formal. Releases have usually been done by me when there have been enough updates that it seems worth advertising and having community members run through an update. In general, anyone could call for and add a PR to make a release, but I would say that so far there haven't been as many/as significant changes as we usually waited for. E.g. v.1.4 included 8 new previewers and the update to signed URLs. That was perhaps an overly long wait, but the community had access to all of this via the betatest folder prior to the official release.

**Comment by sbarbosadataverse** [comment]
@qqmyers Hey Jim, Are you planning on releasing this previewer? There is no labeling or project attached to this issue so it keeps coming up in our biweekly git review. Thanks

**Comment by qqmyers** [comment]
I'm not planning on a release any time soon. The previewer is usable now either from the betatest folder or by downloading and running the previewer locally.

**Comment by cmbz** [comment]
2024/08/19
- Placed on hold. @sbarbosadataverse will monitor and we will revisit during the next Dataverse monthly meeting

**Comment by cmbz** [comment]
2024/11/18: Discussed with @qqmyers this evening. Moving to _This Sprint_ again. Whoever picks up the issue, there is no need for the previewer to be updated. Instead, pick up the latest version in the "betatest folder or the URL https://gdcc.github.io/dataverse-previewers/previewers/betatest/VideoPreview.html" as Jim mentions here: https://github.com/IQSS/dataverse.harvard.edu/issues/283#issuecomment-2187315383
